.gitignore

```diff
@@ -1,28 +0,0 @@
-# Binaries for programs and plugins
```

Comment:
No, they are not long gone. They are still part of the PR

Reply:
At first I remembered you asked me on previous PRs not to add the .gitignore and .vscode/launch.json to the project code, so it got copied from upstream (so I guess after all these files are required to exists)
The I wanted to remove them myself from the project, and that's what I meant